distributed/deploy/tests/test_local.py

```diff
@@ -1061,3 +1063,17 @@ async def test_cluster_names():
 
         async with LocalCluster(processes=False, asynchronous=True) as unnamed_cluster2:
             assert unnamed_cluster2 != unnamed_cluster
+
+
+@pytest.mark.asyncio
+async def test_local_cluster_redundant_kwarg():
```

Comment:
Could you add the `cleanup` fixture here to make sure threads, processes, `rpc`s, etc. are cleaned up

```suggestion
async def test_local_cluster_redundant_kwarg(cleanup):
```